app/controllers/alaveteli_pro/classifications_controller.rb

```diff
@@ -5,8 +5,11 @@ class AlaveteliPro::ClassificationsController < AlaveteliPro::BaseController
   include Classifiable
 
   def create
-    new_status = classification_params[:described_state]
```

Comment:
This commit fixes https://github.com/mysociety/alaveteli-professional/issues/484